presto-docs/src/main/sphinx/optimizer/pushdown.rst

```diff
@@ -15,22 +16,21 @@ The results of this pushdown can include the following benefits:
 Support for pushdown is specific to each connector and the relevant underlying
 database or storage system.
 
-Analysis and Confirmation
--------------------------
+Aggregation Pushdown
+--------------------
 
-Pushdown depends on a number of factors:
+Aggregation pushdown depends on a number of factors:
 
-* generic support for pushdown for that function in Presto
-* function or operation specific support for pushdown in the connector
-* query that allows the detection of the function to push down
-* function needs to exist in the underlying system so it can process the
-  pushdown
+* general support for aggregation pushdown in the connector
+* function-specific support for pushdown in the connector
+* query that allows the pushdown to take place
```

Comment:
```suggestion
* If aggregation pushdown is generally supported by the connector.
* If pushdown of specific function is supported by the connector.
* If query structure allows pushdown to take place.
```

Reply:
I would not do that unless you also change the sentence in front of that list .. with this change it would be wrong

Reply:
It reads fine to me - but I assume you are right :)
As for enumeration, the google documentation style guide recommends making each of the points a capitalized sentence ending with a dot.

Reply:
Mostly true .. so yes, maybe update the opening sentence as well and then go with this

Reply:
Like I said above, I do not see why the current one is wrong (it does not mean it is ok). Can you propose sth which makes this correct English paragraph :)

Reply:
Aggregation push down relies on the following conditions to be true:

or 

Aggregation push down relies on the following conditions to be fulfilled:

Reply:
And you would have to do

* If aggregation pushdown is generally supported by the connector.
* If pushdown of the specific function is supported by the connector.
* If the query structure allows pushdown to take place.

Reply:
Thank you :)